### ISSUE TITLE
Display Active IP Addresses

### ISSUE THREAD

**Issue by AnimatedAneurin** [EvotecIT/PowerBGInfo#7]
So I have yet to try out PowerBGInfo, but was curious to know if Displaying Active IP Addresses is already part of/or going to be part of PowerBGInfo.

Tried quickly looking through your documentation but could not find anything on it (May have missed it so I apologise if this is a waste of your time)

If not already implemented would it be possible to implement something like the below code?

Get-WmiObject -Query "SELECT IPAddress FROM Win32_NetworkAdapterConfiguration WHERE IPEnabled = 'True'" | Select-Object -ExpandProperty IPAddress | Where-Object {$_ -like '...'}

This is something we heavily used in BGInfo as End Users struggle to use CMD to provide IP Addresses to their IT Support. Only needed when users are remote workers using VPN and the DNS Server hasn't updated their records.

**Comment by PrzemyslawKlys** [comment]
I guess you figured out that you can create your own data with `New-BGInfoValue` right?

**Comment by Cool34000** [comment]
Hi,
Try:
```
Get-NetIPAddress | Where-Object { $_.AddressFamily -eq "IPv4" -and $_.IPAddress -ne "127.0.0.1" -and -not ($_.IPAddress -match '^169\.254\.')} | | ForEach-Object {
        $interfaceName = $_.InterfaceAlias
        $interfaceAddr = $_.IPAddress
        switch ($_.PrefixOrigin) {
            "Dhcp"   { $interfaceConf = "(DHCP)" }
            "Manual" { $interfaceConf = "(Fixed IP)" }
        }
        New-BGInfoValue -Name "IP Address ($interfaceName)" -Value "$interfaceAddr $interfaceConf" -Color Black -FontSize 20 -FontFamilyName 'Calibri'
    }

```

**Comment by PrzemyslawKlys** [comment]
It's already added as part of builtin values IPv4Address and IPV6Address

**Comment by Cool34000** [comment]
I know... But I like to have more informations:

![Image](https://github.com/user-attachments/assets/65dd2f9e-9e75-4987-a86f-4190e8c286e1)

That's why I really like PowerBGInfo over BGInfo: it's PowerShell based!

**Comment by PrzemyslawKlys** [comment]
Ye cool, i guess it could be expanded with that, so when someone chooses IPAddress builtin value it shows like your filtered values, but this would require few more adjustments, but it would make life easier for people.

**Comment by Cool34000** [comment]
The default builtin value doesn't fit my need so I created mine.
But that's the power of your module: one can use his own logic to gather and show informations in the way he wants!
I think the default builtin check is good enough for most people... The others can just make their own!

I also don't use OSLastBootUpTime as it returns the date in English format (I'm French so the date is Day/Month/Year and not Month/Day/Year)
So I used:
`$BootTime = ((Get-CimInstance Win32_OperatingSystem).LastBootUpTime).ToString("dd/MM/yyyy 'à' HH:mm")`

I love PowerShell!
